components/api/src/main/java/com/hotels/styx/api/ContentOverflowException.java

```diff
@@ -16,7 +16,12 @@
 package com.hotels.styx.api;
 
 /**
- * An exception that indicates that a content-aggregation operation exceeded the maximum permitted amount of content.
+ * Thrown when HTTP content aggregation fails because received more data than excepted.
+ * <p>
+ * A streaming HTTP message can  be aggregated and converted (aggregated) to a
+ * full message. Normally the aggregator methods ({@code HttpRequest.toFullRequest}, {@code HttpResponse.toFullResponse})
```

Comment:
I don't think the `@code` syntax is correct here. Doesn't it have to be a `#` separating the method name from the class instead of a `.`?

Reply:
I'm thinking to reword this description as:

```
/**
 * Indicates a content aggregation failure because more data was received than expected.
 * <p>
 * A streaming HTTP message can be aggregated to a full message. 
 * The aggregator methods {@link HttpRequest#toFullRequest}, and
 * {@link HttpResponse#toFullResponse} consume data from a network socket or
 * some other unpredictable source and emit this exception when more data is 
 * received than expected.
 */
```